### PR TITLE
February update

### PR DESCRIPTION
**Pull request by dholbach**
We are expecting to land this 2023-03-01 (Wed).

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

File: content/en/blog/2023-03-01-february-update/index.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
```
Comment (dholbach) [inline content/en/blog/2023-03-01-february-update/index.md:88]: @chanwit What do you want to say here for February?

File: content/en/blog/2023-03-01-february-update/index.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+#### Terraform-controller
+
```
Comment (dholbach) [inline content/en/blog/2023-03-01-february-update/index.md:90]: @chanwit What do you want to say here for February?

File: content/en/blog/2023-03-01-february-update/index.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+#### Weave GitOps
+
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.17.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.16.0>
+
```
Comment (dholbach) [inline content/en/blog/2023-03-01-february-update/index.md:95]: @jamwils @davidstauffer What do you want to say here for February?

File: content/en/blog/2023-03-01-february-update/index.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
```
Comment (dholbach) [inline content/en/blog/2023-03-01-february-update/index.md:99]: @juozasg What shall we put in here for February?

File: content/en/blog/2023-03-01-february-update/index.md [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+#### New additions to the Flux Ecosystem
+
+## Recent & Upcoming Events
+
```
Comment (dholbach) [inline content/en/blog/2023-03-01-february-update/index.md:103]: @vanessaabankwah @staceypotter What shall we put in here for Jan/Feb/Mar?

File: content/en/blog/2023-03-01-february-update/index.md [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+sure to mention it in the next post!
+
+### Upcoming Events 📆
+
+We are happy to announce that we have a number of events coming up in
+March - tune in to learn more about Flux and GitOps best practices,
+get to know the team and join our community.
+
+### Flux Bug Scrub
+
```
Comment (dholbach) [inline content/en/blog/2023-03-01-february-update/index.md:125]: @kingdonb Anything we should put in here apart from the updated dates?

File: content/en/blog/2023-03-01-february-update/index.md (hunk 5 above)
Comment (vanessaabankwah) [reply content/en/blog/2023-03-01-february-update/index.md:107]: [2nd March: GitOps Testing in Kubernetes with Flux & Testkube](https://www.meetup.com/gitops-community/events/291670250?utm_medium=referral&utm_campaign=share-btn_savedevents_share_modal&utm_source=link) 

[9th March: Microservices and Kubernetes](https://community.cncf.io/events/details/cncf-cncf-online-programs-presents-cncf-on-demand-webinar-the-path-to-cloud-adoption-and-app-modernization/)

[15th March: CNCF Live Stream -Automating Kubernetes Deployments](https://community.cncf.io/e/mbmpq8/)

[16th March: CNCF On Demand- Kubernetes in 2023 w/ Stefan Prodan & Brendan Burns](https://community.cncf.io/events/details/cncf-cncf-online-programs-presents-cncf-on-demand-webinar-kubernetes-in-2023/)

23rd March: Microsoft Live Stream: Automating Kubernetes Deployments

Comment (dholbach) [reply content/en/blog/2023-03-01-february-update/index.md:107]: Sweet!

File: content/en/blog/2023-03-01-february-update/index.md (hunk 6 above)
Comment (kingdonb) [reply content/en/blog/2023-03-01-february-update/index.md:129]:
```suggestion
The Flux Bug Scrub has undertaken a slightly new format, where we'll be spending more time working on our own code, and even building some new software under the aegis of the "flux-community" organization! We'll still have our familiar spreadsheet every week, but we will be conversely spending less time per-meeting on curating the list of issues across the FluxCD org than we have been, which was the primary focus of these meetings before, and spending more time on code, and discussing cross-cutting issues that might be important for Flux releases (or other objectives that Flux maintainers have on their plates on a day-to-day basis.)

We are also evaluating the possibility of a new, more inclusive "Late Late" Bug Scrub, which would aim to be more accessible to people in the Asia/Pacific time zones, Australia, and even those in Eastern Europe who might not be able to make these earlier times. If you are in these time zones, it'll actually be "Early Work Hours" if we've calibrated this correctly. The Bug Scrub community is small but growing; we need your feedback to make this a hit (which you can provide asynchronously in the Flux channel on CNCF slack, please remember to @ KingdonB if you are interested!)

The addition of another meeting at a new time of day, is intended to provide a little extra time for the curation of issues to go along with this new format change, and also to balance the scales somewhat regionally (which currently, the timing favors Flux contributors or new joiners who are using Flux at their day jobs in both the Americas and Europe.)
```

**Comment by stefanprodan** [comment]
@dholbach please include https://truelayer.com/blog/flux2-migration-how-we-dropped-cpu-usage-by-nearly-40x

File: content/en/blog/2023-03-01-february-update/index.md (hunk 6 above)
Comment (kingdonb) [reply content/en/blog/2023-03-01-february-update/index.md:129]: I gave the above a round of edits and I think it flows much better, more focused on the most interesting points. I'm going to go ahead and commit it, but feel free to edit further.

**Comment by staceypotter** [comment]
@dholbach can we also add this upcoming tech talk by Julian Hennig (https://github.com/jlnhnng) from Mirantis?
[GitOps: Automatic Deployments and Updates with Flux (Tue. March 7 @ 6am ET | 10am GMT | 11am CET)](https://www.mirantis.com/labs/gitops-automatic-deployments-and-updates-with-flux/)

File: content/en/blog/2023-03-01-february-update/index.md [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
+#### Terraform-controller
+
```
Comment (chanwit) [inline content/en/blog/2023-03-01-february-update/index.md:94]:
```suggestion

The latest release of TF-Controller, version v0.14.0, introduces several new features and many bug fixes. Notably, the release offers first-class support for Terraform Cloud with the `spec.cloud` field. This enhancement allows Weave GitOps Enterprise users to leverage GitOps Templates with Terraform Cloud as a backend for their Terraform resources, opening up a world of possibilities for GitOps workflows.

In addition to Terraform Cloud support, the update upgrades Flux to v0.40.0 and Terraform to v1.3.9, with bug fixes including improved AWS package documentation, and missing inventory entries.

The new release also offers multi-arch image support, customizable controller log encoding, and the option to configure Kube API QPS and Burst. The Terraform apply stage now features a parallelism option for even more customization.

Users are highly recommended to upgrade to TF-Controller v0.14.0 to take advantage of these improvements. For any feedback or questions, please reach out to the team on [the GitHub repository](https://github.com/weaveworks/tf-controller).
```

File: content/en/blog/2023-03-01-february-update/index.md [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
```
Comment (chanwit) [inline content/en/blog/2023-03-01-february-update/index.md:92]:
```suggestion

The team has recently updated Flamingo by rebasing it onto the upstream ArgoCD versions v2.3.17, v2.4.23, and 2.5.11. This update has been made in response to the recent vulnerability CVE-2023-23947, and the team strongly recommends that all users update their systems as soon as possible.

Updated Flamingo images are:
  - v2.3.17-fl.3-main-bc5b4abb
  - v2.4.23-fl.3-main-bc5b4abb
  - v2.5.11-fl.3-main-bc5b4abb
```

File: content/en/blog/2023-03-01-february-update/index.md (hunk 4 above)
Comment (dholbach) [reply content/en/blog/2023-03-01-february-update/index.md:126]: I'm aiming for a release of this update on Wednesday.

File: content/en/blog/2023-03-01-february-update/index.md [hunk 9]
```diff
@@ -0,0 +1,425 @@
...
+#### Pulumi Kubernetes Operator
+
+Michel Bridgen wrote a blog post about [how to combine Pulumi with
+Flux](/blog/2023/02/flux-pulumi-superpowers/) using the Pulumi Kubernetes
+Operator, which extends the reach of both Flux and Pulumi.
+
+What you can look forward to in the next release of the operator is that -
+based on Paolo's work on git-go - the operator (and Pulumi itself) [will be able
+work with Azure DevOps](https://github.com/pulumi/pulumi/pull/12001).
+
```
Comment (dholbach) [inline content/en/blog/2023-03-01-february-update/index.md:203]: @squaremo Does this look good to you?

File: content/en/blog/2023-03-01-february-update/index.md (hunk 4 above)
Comment (juozasg) [reply content/en/blog/2023-03-01-february-update/index.md:193]: Version 0.23.0 of vscode-gitops-tools extension was released. This version introduces a new webview for for configuring GitRepository, HelmRepository, OCIRepository, Bucket and Kustomization resources. Extension context (right-click) actions now work with multiple open repositories in the expected way.

**Comment by dholbach** [comment]
→ https://deploy-preview-1365--fluxcd.netlify.app/blog/

**Pull request merged**